lib/recipient.rb

```diff
@@ -0,0 +1,46 @@
+module Slack
+  class SlackApiError < StandardError; end
+
+  class Recipient
+    attr_reader :slack_id, :name
+
+    def initialize(slack_id, name)
+      @slack_id = slack_id
+      @name = name
+    end
+
+    BASE_URL = "https://slack.com/api/"
+
+    def self.get(url, params)
+      return HTTParty.get(url, query: params)
+    end
```

Comment:
I think the `params` argument for this method is always the same: a hash containing the slack token. Could you supply that here and remove the extra argument?

Also, you should be doing things like error handling here.

In general, every time you add another layer to a piece of functionality, you should take care of some part of the puzzle in that layer. In this case, `Channel.list` is in charge of turning a response into an array of `Channel` objects, `Channel.channels_get` knows the URL and the query parameters, but this method doesn't add anything new.